lib/libzrepl/data_conn.c

```diff
@@ -379,9 +393,11 @@ uzfs_zvol_worker(void *arg)
 	}
 
 	/*
-	 * We are not sending ACK for writes meant for rebuild
+	 * We are not sending ACK for writes/unmap meant for rebuild
 	 */
-	if (rebuild_cmd_req && (hdr->opcode == ZVOL_OPCODE_WRITE)) {
+	if (rebuild_cmd_req && ((hdr->opcode == ZVOL_OPCODE_WRITE) ||
+	    ((hdr->opcode == ZVOL_OPCODE_UNMAP) &&
+	    ZVOL_IS_REBUILDING(zinfo->main_zv)))) {
```

Comment:
this check is not required right?

Reply:
yes.. it can be removed since `rebuild_cmd_req` check is also there.